pages/querying/clauses/alter.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: ALTER clause
+description: Understand the ALTER clause in Memgraph's operations.
+---
+
+import { Callout } from 'nextra/components'
+
+# ALTER clause
+
+The `ALTER` clause is used to alter existing definition of enums.
```

Comment:
```suggestion
The `ALTER` clause is used to alter existing definitions of enums."
```

Reply:
Why `"`?

Reply:
I'm ok with plural `definitions` and plural `enums`.
I was questioning the `"` at the end.

Reply:
Oh, sorry... :( That is 100% a typo on my part. Thank you for catching it!